crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -70,16 +72,20 @@ pub struct SlidingSyncList {
     /// Name of this list to easily recognize them
     pub name: String,
 
-    /// The state this list is in
+    /// The state this list is in.
     state: Arc<StdRwLock<Observable<SlidingSyncState>>>,
 
-    /// The total known number of rooms,
-    rooms_count: Arc<StdRwLock<Observable<Option<u32>>>>,
+    /// The total number of rooms that is possible to interact with for the
+    /// given list. It's not the total rooms that have been fetched. The
+    /// server tells the client that it's possible to fetch this amount of
+    /// rooms maximum. Since this number can change according to the list
+    /// filters, it's observable.
```

Comment:
Would be nice to split this into at least two paragraphs (short + long explanation, like commit message and extended description).

Reply:
Done.